src/ophyd_async/panda/panda.py

```diff
@@ -242,6 +228,26 @@ def set_attribute(self, name: str, num: Optional[int], block: Device):
         else:
             setattr(self, name, block)
 
+    async def pvi_get(self, prefix: str) -> Dict[str, PVIEntry]:
+        """Makes a PvaSignalBackend purely to connect to PVI information.
+
+        This backend is simply thrown away at the end of this method. This is useful
+        because the backend handles a CancelledError exception that gets thrown on
+        timeout, and therefore can be used for error reporting."""
+
+        read_pv = prefix + "PVI:"
+        backend: SignalBackend = PvaSignalBackend(None, read_pv, read_pv)
+        await backend.connect()
+        pv_info: Dict[str, Dict[str, str]] = await backend.get_value()
+
+        result = {}
+
+        for attr_name, attr_info in pv_info.items():
+            result[attr_name] = PVIEntry(**attr_info)  # type: ignore
+
+        _remove_inconsistent_blocks(result)
+        return result
```

Comment:
Please can you put all the `pvi` related bits into their own file in `ophyd_async.epics.pvi` please? These will be used on things that are not PandA in the future

Reply:
Assuming that remove_inconsistent_blocks is Panda specific enough to stay, I've moved pvi_get, make_signal, PVIEntry, and the _pvi_mapping.